### ISSUE TITLE
wrong extra axis lines

### ISSUE THREAD

**Issue by 2sn** [nschloe/matplotlib2tikz#196]
ipython/pylab

```
Python 3.6.1 (default, Mar 23 2017, 07:46:13) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.1.0 -- An enhanced Interactive Python. Type '?' for help.
<IPython.terminal.interactiveshell.TerminalInteractiveShell object at 0x7fef2936a588>
Using matplotlib backend: TkAgg
Populating the interactive namespace from numpy and matplotlib
```

I then use interactively

```python
plot([0,1])
xlabel(r'$x$')
ylabel('$y$')
from matplotlib2tikz import save
save('text.tex')
```

This creates the following output file

```latex
% This file was created by matplotlib2tikz v0.6.11.
\begin{tikzpicture}

\definecolor{color0}{rgb}{0.12156862745098,0.466666666666667,0.705882352941177}

\begin{axis}[
xlabel={$x$},
ylabel={$y$},
xmin=-0.05, xmax=1.05,
ymin=-0.05, ymax=1.05,
xtick={-0.2,0,0.2,0.4,0.6,0.8,1,1.2},
xticklabels={,0.0,0.2,0.4,0.6,0.8,1.0,},
ytick={-0.2,0,0.2,0.4,0.6,0.8,1,1.2},
yticklabels={,0.0,0.2,0.4,0.6,0.8,1.0,},
tick align=outside,
tick pos=left,
x grid style={lightgray!92.026143790849673!black},
y grid style={lightgray!92.026143790849673!black}
]
\addplot [semithick, color0, forget plot]
table {%
0 0
1 1
};
\path [draw=black, fill opacity=0] (axis cs:0,-0.05)
--(axis cs:0,1.05);

\path [draw=black, fill opacity=0] (axis cs:1,-0.05)
--(axis cs:1,1.05);

\path [draw=black, fill opacity=0] (axis cs:-0.05,0)
--(axis cs:1.05,0);

\path [draw=black, fill opacity=0] (axis cs:-0.05,1)
--(axis cs:1.05,1);

\end{axis}

\end{tikzpicture}
```
where the last 4 `\path` command clearly are wrong.  And there seems lots of other superfluous stuff when compared to just using python and importing matplotlib.pylab manually

```
from matplotlib.pylab import *
plot([0,1])
xlabel(r'$x$')
ylabel('$y$')
from matplotlib2tikz import save
save('text.tex')
```
and I get

```latex
% This file was created by matplotlib2tikz v0.6.11.
\begin{tikzpicture}

\definecolor{color0}{rgb}{0.12156862745098,0.466666666666667,0.705882352941177}

\begin{axis}[
xlabel={$x$},
ylabel={$y$},
xmin=-0.05, xmax=1.05,
ymin=-0.05, ymax=1.05,
tick align=outside,
tick pos=left,
x grid style={lightgray!92.026143790849673!black},
y grid style={lightgray!92.026143790849673!black}
]
\addplot [semithick, color0, forget plot]
table {%
0 0
1 1
};
\end{axis}

\end{tikzpicture}
```

**Comment by nschloe** [comment]
Perhaps a dup of #78?

**Comment by 2sn** [comment]
yes, it may be related, though it seems slightly different in detail. 
It appears to be realted to the ipython pylab environment, maybe because 
TkAgg is being loaded?

**Comment by 2sn** [comment]
yes, it may be related, though it seems slightly different in detail. It 
appears to be realted to the ipython pylab environment, maybe because 
TkAgg is being loaded?

**Comment by nschloe** [comment]
Fixed by d3819ba.
